sdk/tables/Azure.Data.Tables/samples/Sample6TransactionalBatch.md

```diff
@@ -35,41 +35,22 @@ A common use case for batch operations is to add many entities to a table in bul
 ```C# Snippet:BatchAdd
 // Create a list of 5 entities with the same partition key.
 string partitionKey = "BatchInsertSample";
-List<TableEntity> entityList = new List<TableEntity>{
-    new TableEntity(partitionKey, "01")
-    {
-        {"Product", "Marker" },
-        {"Price", 5.00 },
-        {"Brand", "Premium" }
-    },
-    new TableEntity(partitionKey, "02")
-    {
-        {"Product", "Pen" },
-        {"Price", 3.00 },
-        {"Brand", "Premium" }
-    },
-    new TableEntity(partitionKey, "03")
-    {
-        {"Product", "Paper" },
-        {"Price", 0.10 },
-        {"Brand", "Premium" }
-    },
-    new TableEntity(partitionKey, "04")
-    {
-        {"Product", "Glue" },
-        {"Price", 1.00 },
-        {"Brand", "Generic" }
-    },
+List<TableEntity> entityList = new List<TableEntity>
+{
+    new TableEntity(partitionKey, "01") { { "Product", "Marker" }, { "Price", 5.00 }, { "Brand", "Premium" } },
+    new TableEntity(partitionKey, "02") { { "Product", "Pen" }, { "Price", 3.00 }, { "Brand", "Premium" } },
+    new TableEntity(partitionKey, "03") { { "Product", "Paper" }, { "Price", 0.10 }, { "Brand", "Premium" } },
+    new TableEntity(partitionKey, "04") { { "Product", "Glue" }, { "Price", 1.00 }, { "Brand", "Generic" } },
 };
 
 // Create the batch.
-TableTransactionalBatch addEntitiesBatch = client.CreateTransactionalBatch(partitionKey);
+List<BatchItem> addEntitiesBatch = new();
```

Comment:
Should we consider using the legacy format for instantiation in samples?  This syntax is new enough that readers may not be familiar with it.